src/types/AppointmentFormData.tsx

```diff
@@ -1,14 +1,25 @@
-type AppointmentFormData = {
-  firstName: string
-  lastName: string
-  phoneNum: string
+export type AppointmentFormData = {
+  first_name: string
+  last_name: string
+  mobile_phone: string
   email: string
   dob: string
-  apptDate: string
-  apptTime: string
-  apptType: string
-  optionalDescription: string
-  emergency: string
+  requested_date: string
+  requested_time: string
+  appointment_type: string
+  description: string
+  is_emergency: string
```

Comment:
I am not sure how this `is_emergency` field works. On the DB it's set as a boolean.
I do think Supabase allows for some type generation from introspection. Maybe we can use that to ensure uniformity with types across the frontend and backend?